imageproxy.go

```diff
@@ -211,6 +225,41 @@ func (p *Proxy) allowed(r *Request) error {
 	return fmt.Errorf("request does not contain an allowed host or valid signature: %v", r)
 }
 
+// allowedContentType returns an allowed content type string to use in responses or "" if the
+// content type cannot be used.
+func (p *Proxy) allowedContentType(contentType string) string {
```

Comment:
Why return a string?  It doesn't look like the return value is really needed.  Instead, perhaps rename this method to validContentType and return a bool like the other valid* methods?

Reply:
I feel like there was a good reason for this, but I can't remember it. I'll refactor if it doesn't come to me.